### ISSUE TITLE
Fix github actions

### ISSUE THREAD

**Issue by Phill93** [SituationBoard/SituationBoard#4]

**Comment by Sebastian-Maier** [comment]
Strange. All GitHub Actions did work (and all tests did pass) until recently. 😕
Seems to be some kind of setup issue that only occurs on latest GitHub runners (with new environments):

```
Current runner version: '2.277.1'
Operating System
  Ubuntu
  20.04.2
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20210309.1
```
vs.
```
Current runner version: '2.277.1'
Operating System
  Ubuntu
  20.04.2
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20210302.0
```

I will look into it as soon as possible.

**Comment by Sebastian-Maier** [comment]
This problem was caused by a combination of two independent issues:
1. Some package repository server issue (see [https://github.com/actions/runner/issues/932](https://github.com/actions/runner/issues/932)):
"The repository 'https://dl.bintray.com/sbt/debian  Release' no longer has a Release file."
2. Some Microsoft EULA was not accepted during the headless "apt-get upgrade" setup step:
"ERROR: The EULA was not accepted. Installation aborted."

To avoid these kinds of problems in the future, we now pass `CI_ACTIVE=1` to all setup steps within GitHub Action workflows and GitLab CI pipelines. If this `CI_ACTIVE` environment variable is set to `1`, `sbctl` avoids certain (optional) setup steps – like upgrading packages – that can cause problems in CI environments. As a bonus, this adjustment could also speed up the execution of the CI tests.

This issue is fixed by commit d9bfd44
